### PR TITLE
Serve js files from available CDNs (connect #1688)

### PR DESCRIPTION
**Pull request by kymni**
#### Before the PR (what is the issue or what needed to be done)
Some JavaScript files were served from Flow
#### The solution
Changed the source of files to available CDNs

## Checklist
* [x] Connect the issue
* [ ] Test plan
* [x] Copyright header
* [x] Code formatting
* [x] Documentation

### CODE REVIEW

File: Dashboard/app/dashboard.html [hunk 1]
```diff
@@ -24,13 +24,13 @@
     <script>
         window.jQuery || document.write("<script src='/vendorjs/js/vendor/jquery-1.8.2.min.js'><\/script>")
     </script>
```
Comment (muloem) [inline Dashboard/app/dashboard.html]: I think we can remove the above lines and also load jquery from cloudflare.

File: Dashboard/app/dashboard.html [hunk 2]
```diff
@@ -24,13 +24,13 @@
...
-    <script src="/vendorjs/js/vendor/handlebars-1.0.rc.1.js"></script>
-    <script src="/vendorjs/js/vendor/ember-1.0.0.pre-2-36.min.js"></script>
+    <script src="//cdnjs.cloudflare.com/ajax/libs/handlebars.js/1.0.rc.1/handlebars.min.js"></script>
+    <script src="//cdnjs.cloudflare.com/ajax/libs/ember.js/1.0.0-pre.2/ember-1.0.0-pre.2.min.js"></script>
     <script src="/vendorjs/js/vendor/ember-data-rev10.min.js"></script>
     <script src="/vendorjs/js/vendor/jquery-ui-1.8.21.custom.js"></script>
```
Comment (muloem) [inline Dashboard/app/dashboard.html]: Isn't there a CDN for this one?

Comment (kymni) [inline Dashboard/app/dashboard.html]: This version of ember-data seems to precede even the earliest tagged version so couldn't find it on any CDN.

**Comment by iperdomo** [comment]
If the resource is available over https, we should always use the https version. 

* https://www.paulirish.com/2010/the-protocol-relative-url/

We can also add `integrity` to those files.

* https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity
* http://caniuse.com/#feat=subresource-integrity

File: Dashboard/app/dashboard.html [hunk 3]
```diff
@@ -20,14 +20,11 @@
 <body class="appPage">
 
     <!-- Grab Google CDN's jQuery. fall back to local if necessary -->
```
Comment (muloem) [inline Dashboard/app/dashboard.html]: We don't need this comment either.

File: Dashboard/app/public.html [hunk 4]
```diff
@@ -28,14 +28,13 @@
   <body class="publicPage">
 
   <!-- Grab from a CDN, fall back to local if necessary -->
```
Comment (muloem) [inline Dashboard/app/public.html]: Ditto for this comment

**Comment by kymni** [comment]
Regarding loading libraries via HTTPS, I've not specified the protocol in the src so the libraries will always be pulled using the same protocol as the dashboard.

As for d3, suggested fix is tied to PR #1953

**Comment by muloem** [comment]
> Regarding loading libraries via HTTPS, I've not specified the protocol in the src so the libraries will always be pulled using the same protocol as the dashboard.

@kymni true. I think the idea is to explicitly specify HTTPS.

**Pull request merged**